src/script/page/AppLock.tsx

```diff
@@ -244,6 +245,12 @@ const AppLock: React.FC<AppLockProps> = ({
     setState(APPLOCK_STATE.NONE);
     setSetupPassphrase('');
   };
+  const cancelAppLock = isAppLockEnforced
+    ? undefined
+    : () => {
+        appLockRepository.setEnabled(false);
+        setIsVisible(false);
+      };
```

Comment:
```suggestion
  const onCancelAppLock = () => {
        appLockRepository.setEnabled(false);
        setIsVisible(false);
      };
```

Reply:
I'd rather not create this function at all if we don't need it.

Reply:
It's doesn't matter, you will have function after code transpilation.  So we don't need another checking if functions exist. Because you running action to check if function exist. :)

Or i'm wrong thinking.